src/client-apps/nuxeo-mobile/how-to-customize-nuxeo-mobile-workflow-tasks-layout.md

```diff
@@ -0,0 +1,333 @@
+
+## Layout Format
+
+The format of the layout is simply a JSON object with the list of task fields to display and properties.
+
+The default layout looks like:
+
+```json
+{
+  "name": "task id",
+  "layout": [
+    {
+      "metadata": [
+        {
+          "label": "label name",
+          "field": "studio configured field",
+          "type": "cf available fields...",
+          "properties": {
+            "property1": "value",
+            "etc."
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+The list of types is detailed below with an example. 
+ 
+
+## Steps
+
+### Add a JSON resource
+In the **Resources** tab of the Designer, tab, click on the workflow and click on “create” to add an empty file with the name of the related task and the json format :
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/CLIENT-APPS/Mobile/HOW_TO_workflows_studio_create_json
+    name: HOW_TO_workflows_studio_create_json.png
+    1.13#screenshot#up_to_date
+--}}
+![HOW_TO_workflows_studio_create_json](nx_asset://ae86da20-cd15-4327-b2b6-a89b4c081f48 ?w=400)
+
+
+
+Note that the name of this file should match the task .html file name.
+
+Add the following content to this file to enable the task to display the directory selector : 
+```json
+{
+  "name": "nuxeo-task879",
+  "layout": [
+    {
+      "metadata": [
+        {
+          "label": "Type of contract",
+          "field": "type",
+          "type": "directory",
+          "properties": {
+            "multivalued": false,
+            "required": true,
+            "placeholder": "Select the contract type...",
+            "directoryName": "type",
+            "type":"select"
+            
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+Save then deploy the package.
+
+### Test your configuration on mobile
+Open Nuxeo Mobile app and set your local url. (can be found on ip tools according to your OS. ex : 172.16.32.75:8080)
+
+Login as administrator ; Assuming you created a document of type “contract” (again, the previous tutorial should be completed), go to this document on mobile and click on the overflow button > start process.
+
+A task is affected to the Administrator user. When opening this task, it should look likes this :
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/CLIENT-APPS/Mobile/HOW_TO_workflows_task_result
+    name: HOW_TO_workflows_task_result.png
+    1.13#screenshot#up_to_date
+--}}
+![HOW_TO_workflows_task_result](nx_asset://5c41d39e-45bd-4272-aa46-58113d4dd1f6 ?w=500)
+
+## Properties list 
+
+Properties can be added depending on the task type :
+
+#### For every tasks type
+```json
+“readonly”:[true/false]
+```
+Used to display informations without letting the user change its value.
+
+
+
+```json
+“required” : [true, false] 
```

Comment:
Wrong “ and “, it should be " as in the JSON layout above otherwise, copy/paste won't work.
Change to be done in the whole file.